meteor/client/ui/Status/DebugState.tsx

```diff
@@ -0,0 +1,71 @@
+import * as React from 'react'
+import { Translated } from '../../lib/ReactMeteorData/react-meteor-data'
+import * as reacti18next from 'react-i18next'
+
+interface IDebugStateTableProps {
+	debugState: object
+}
+interface IDebugStateTableState {}
+
+export const DebugStateTable = reacti18next.withTranslation()(
```

Comment:
I would prefer this to be a functional/hooks component, but that doesnt need to be done

Reply:
Yeah, me too, but I don't think that's a reason to block this.